pkg/payload/render.go

```diff
@@ -76,8 +78,8 @@ func renderDir(renderConfig manifestRenderConfig, idir, odir string, skipFiles s
 		if skipFiles.Has(file.Name()) {
 			continue
 		}
-		if strings.Contains(file.Name(), "CustomNoUpgrade") || strings.Contains(file.Name(), "TechPreviewNoUpgrade") {
-			// CustomNoUpgrade and TechPreviewNoUpgrade may add features to manifests like the ClusterVersion CRD,
+		if strings.Contains(file.Name(), "CustomNoUpgrade") || strings.Contains(file.Name(), "TechPreviewNoUpgrade") || strings.Contains(file.Name(), "DevPreviewNoUpgrade") {
+			// CustomNoUpgrade, TechPreviewNoUpgrade and DevPreviewNoUpgrade may add features to manifests like the ClusterVersion CRD,
```

Comment:
Thanks for tackling this post-bump installation issue :+1: . We ran into it in https://github.com/openshift/cluster-version-operator/pull/1045 but did not have space to investigate it.